doc/mf6io/prt/prt.tex

```diff
@@ -21,6 +21,48 @@ \subsection{Specifying Cell Face Flows using IFLOWFACE}
 \subsection{Particle Mass Budget}
 A summary of all inflow (sources) and outflow (sinks) of particle mass is called a mass budget.  The particle mass budget is printed to the PRT Model Listing File for selected time steps.  In the current implementation, each particle is assigned unit mass, and the numerical value of the flow can be interpreted as particles per time.
 
+\subsection{Vertical Tracking}
```

Comment:
Should this be here or in the PRP chapter? Also, should it be restated in the supplemental technical info?

..planning to repackage some of this with illustration in an informal migration guide as well, to distribute with the release to testing partners / users

Reply:
Agree.  Though in supptechinfo I'd favor a very generic description that points to mf6io for specifics, since we'll likely be refining the dry tracking options down the road